### ISSUE TITLE
overlaysWebView change the overlay not only status bar but the keyboard also

### ISSUE THREAD

**Issue by ryakoviv** [apache/cordova-plugin-statusbar#103]
After calling StatusBar.overlaysWebView(true); keyboard also cover webview instead of resize it

**Comment by janpio** [comment]
Platform?
Versions of everything? (Cordova, Cordova Platform, Plugin, OS, Device...)

**Comment by spinninghamster** [comment]
Can you provide a screenshot?

**Comment by breautek** [comment]
I believe this might be related to #110

**Comment by tharunkumar0** [comment]
@breautek yeah but any solutions?

**Comment by breautek** [comment]
@tharunkumar0 No, I spent a few hours playing with it, but I don't have much experience in native android development.

Here is what I do know...

The view gets a FULLSCREEN attribute, which appears to be necessary to get the overlay status bar, but it also messes up the keyboard functionality. The view has softInputMode of ADJUST_RESIZE, which tells the view to resize when the keyboard is visible. The problem is, ADJUST_RESIZE isn't compatible with the FULLSCREEN flag. So while FULLSCREEN flag is set, the keyboard will just appear above the view.
